integration_tests/pages/page.ts

```diff
@@ -121,8 +121,8 @@ export default abstract class Page {
     cy.get('a').contains(label).click()
   }
 
-  shouldShowPrintButton(): void {
-    cy.get('button').contains('Save as PDF')
+  shouldShowPrintButton(text = 'Save as PDF'): void {
```

Comment:
probably more a question for Chanel/Alex, is there a reason why we show 'Save as PDF' on one view and 'Download as a PDF' on another?

Reply:
Yeah, it's a good point. I'll confirm before merging 👍🏾